src/test-utils/dom/date-range-picker/index.ts

```diff
@@ -100,12 +100,25 @@ export class DrpDropdownWrapper extends ComponentWrapper {
     return this.findByClassName(styles['calendar-header'])!;
   }
 
-  findPreviousMonthButton(): ButtonWrapper {
-    return this.findComponent(`.${styles['calendar-prev-month-btn']}`, ButtonWrapper)!;
+  findPreviousPageButton(): ButtonWrapper {
+    return this.findComponent(`.${styles['calendar-prev-page-btn']}`, ButtonWrapper)!;
```

Comment:
⛳️ This change will break consumers' tests. 

To make this damage minimal, we extract such classes into `test-classes/styles.scss` file. This should be done for all class names changed in this file